packages/react-native-platform-override/src/GitReactFileRepository.ts

```diff
@@ -29,32 +30,32 @@ export default class GitReactFileRepository
   implements VersionedReactFileRepository {
   private gitClient: simplegit.SimpleGit;
   private gitDirectory: string;
-  private checkedOutVersion: string;
+  private checkedOutVersion?: string;
 
   // We have a potential race condition where one call to getFileContents
   // could checkout out a new tag while an existing call is rading a file.
   // Queue items to ensure the read operation is performed atomically
   private actionQueue: ActionQueue;
 
-  private constructor() {}
+  private constructor(gitDirectory: string, gitClient: simplegit.SimpleGit) {
+    this.actionQueue = new ActionQueue();
```

Comment:
Changed bc TS strict settings enforces that we set instance variables in the constructor or mark them possibly undefined.